velox/exec/tests/TableWriteTest.cpp

```diff
@@ -63,6 +63,8 @@ std::string testModeString(TestMode mode) {
       return "PARTITIONED";
     case TestMode::kBucketed:
       return "BUCKETED";
+    default:
+      return "UNKNOWN";
```

Comment:
Is this expected - why not throw a VELOX_NOT_SUPPORTED exception.

Reply:
Will update, thanks.

Reply:
Updated.